Samples/LineGraph/README.md

```diff
@@ -10,18 +10,25 @@ In order to use the library, you have to install it. This is done by running the
 uno install Fuse.Charting
 ```
 
-With that done, you should be all set.
+With that done, you should be all set. Remember to include `Fuse.Charting` package in your unoproj file, like it is done in this example:
+
+```
+"Packages": [
+    [...]
+    "Fuse.Charting"
+  ],
+```
 
 ## The line-graph renderer
 
-The line-graph renderer should function as a `ux:Class` we can place inside a `Plot` element to have a graph drawn the way we want it. We are therefore interrested in adding a few properties so we get more control over the graph:
+The line-graph renderer should function as a `ux:Class` we can place inside a `Plot` element, acting as a shortcut to a styled line graph. We are therefore interested in adding a few properties so we get more control over the graph:
```

Comment:
I was more confused by the "should function as a `ux:Class`" here. Try to be less abstract